trunk/SUAVE/Methods/Performance/V_n_diagram.py

```diff
@@ -0,0 +1,990 @@
+    S. Gudmundsson "General Aviation Aircraft Design: Applied Methods and Procedures", Butterworth-Heinemann; 1 edition
+
+    Inputs:
+    V_n_data.
+        airspeeds.positive                  [kts]
+            negative                        [kts]
+        load_factors.positive               [Unitless]
+            negative                        [Unitless]
+        weight                              [lb]
+        density                             [slug/ft**3]
+        density_ratio                       [Unitless]
+        lift_coefficient                    [Unitless]
+        reference_area                      [ft**2]
+    lower_bound                             [Unitless]
+    Num_of_points                           [Unitless]
+    sign_flag                               [Unitless]
+
+    Outputs:
+    V_n_data.
+        load_factors.positive               [Unitless]
+            negative                        [Unitless]
+        airspeeds.positive                  [kts]
+            negative                        [kts]
+
+    Properties Used:
+    N/A
+
+    Description:   
+    """  
+    # Unpack
+    weight          = V_n_data.weight
+    reference_area  = V_n_data.reference_area
+    density_ratio   = V_n_data.density_ratio
+    rho             = V_n_data.density
+    
+    if sign_flag == 1:
+        load_factors     = V_n_data.load_factors.positive
+        airspeeds        = V_n_data.airspeeds.positive
+        lift_coefficient = V_n_data.maximum_lift_coefficient
+
+    elif sign_flag == 2:
+        load_factors     = V_n_data.load_factors.negative
+        airspeeds        = V_n_data.airspeeds.negative
+        lift_coefficient = V_n_data.minimum_lift_coefficient
+    
+    delta = (airspeeds[upper_bound] - airspeeds[lower_bound]) / (Num_of_points + 1)     # Step size
+    for i in range(Num_of_points):       
+        coef      = lower_bound + i + 1
+        airspeeds = np.concatenate((airspeeds[:coef], [airspeeds[lower_bound] + (i + 1) * delta], airspeeds[coef:]))
+        Vtas      = airspeeds[coef] / density_ratio * Units.knots / Units['ft/s']      
+        if load_factors[1] > 0:
+            nl = 0.5 * rho * Vtas**2 * reference_area * lift_coefficient / weight
+        else:
+            nl = -0.5 * rho * Vtas**2 * reference_area * abs(lift_coefficient) / weight
+            
+        load_factors = np.concatenate((load_factors[:coef], [nl], load_factors[coef:]))
+
+    # Pack
+    if sign_flag == 1:
+        V_n_data.load_factors.positive     = load_factors
+        V_n_data.airspeeds.positive        = airspeeds
+
+    elif sign_flag == 2:
+        V_n_data.load_factors.negative     = load_factors
+        V_n_data.airspeeds.negative        = airspeeds
+    
+    return 
+#--------------------------------------------------------------------------------------------------------------
+
+def gust_loads(category_tag, V_n_data, Kg, CLa, Uref_rough, Uref_cruise, Uref_dive, Num_of_points, FAR_part_number, sign_flag):
```

Comment:
As @planes already suggested, since you are passing lots of parameters, perhaps consider passing a data structure:
gust_data           = Data()
gust_data.airspeeds = airspeeds
etc....

Reply:
Sure, I will definitely group the gust speeds. However, grouping other parameters will not make sense